pkg/controller/apimanager/apimanager_controller.go

```diff
@@ -88,6 +89,11 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	err = c.Watch(&source.Kind{Type: &v1beta1.PodDisruptionBudget{}}, ownerHandler)
```

Comment:
What is the motivation behind watching PDB resources?

Reply:
The idea is that whenever changes are made to the PDBs, it will trigger a reconcile action on the operator so that it will resync them if there are changes made by the user. 

Happy to remove it if you guys think it is unnecessary, as at the moment, pre k8s 1.16, it is not possible to update PDBs once they are created.  However, from k8s 1.16 (openshift 4.3 I believe), it will be possible to update PDBs. Generally speaking though, I think it is a good practice to watch for all the secondary resources that are managed by the operator to ensure they are kept in sync.

Reply:
I think it's a good idea to keep the Watch in the secondary resources so I'd keep it. That's why the other secondary resources were added in the code too (to react to changes and keep them in the desired state)

Reply:
The operator does not have any action implemented to react upon PDB resource updates. I would remove this watch, or we will have unneeded reconciliation loops.

wdty @miguelsorianod ?

Reply:
@eguzki I am updating this PR now, there will be logic to update/remove PDBs based on the suggestion from @miguelsorianod. Will ping you once it's ready.

Reply:
Ok, there is a point. I did not get it the first time. If someone updates PDB manually, the operator will update them back to desired state. Fair enough.

Thanx

Reply:
@eguzki, even if we don't make specific fields configurable at CR level we are interested in keeping the desired state of our model with what currently exists in Kubernetes.

You want to react to events that happen in the resource to keep it in sync with the desired state, even if it is not configurable at CR level. It's a very common pattern among controllers.
Let's keep the watch.